Bottom: 4370cb1efb3b3895d7330a6de29b0346df1d69b9
Top:    1c64168649b10f7968837df55a9274d0ae944b12
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-05 21:05:50 +0100

hid/gtk: Enable transparent lines in GL renderer


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index d9ee158..7916b41 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -79,7 +79,7 @@ ghid_set_layer (const char *name, int group, int empty)
 
   if (idx >= 0 && idx < max_copper_layer + 2)
     {
-      priv->trans_lines = false;
+      priv->trans_lines = true;
       return PCB->Data->Layer[idx].On;
     }
   if (idx < 0)
